configs/orchestrator/templates/general-config.yaml

```diff
@@ -4,5 +4,41 @@ metadata:
   name: general-config
   namespace: orchestrator
 data:
-  # Add more non-secret env vars here.
   CLOUD: {{ .Values.benchmarkingClusterCloud }}
+
+  # AWS specific values.
+  {{if .Values.clouds.aws.defaultRegion}}
+  AWS_DEFAULT_REGION: {{ .Values.clouds.aws.defaultRegion }}
+  {{end}}
+  {{if .Values.clouds.aws.benchmarkWorkerCount }}
+  BENCHMARK_WORKER_COUNT: {{ .Values.clouds.aws.benchmarkWorkerCount | quote }}
+  {{end}}
+  {{if .Values.clouds.aws.benchmarkInstanceType }}
+  BENCHMARK_INSTANCE_TYPE: {{ .Values.clouds.aws.benchmarkInstanceType }}
+  {{end}}
+  {{if .Values.clouds.aws.benchmarkOSArch }}
+  BENCHMARK_OS_ARCH: {{ .Values.clouds.aws.benchmarkOSArch }}
```

Comment:
In my exp with OCI, we don't need to provide the arch field with workers because depending on the arch of the machine the correct docker images are pulled and the kubelet and other things come up fine without issue.